### PR TITLE
[16.0][MIG] delivery_cttexpress: Migration to 16.0

### PR DESCRIPTION
**Pull request by chienandalu**
Standard migration

cc @Tecnativa TT40505 

please review @victoralmau

### CODE REVIEW

File: delivery_cttexpress/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "name": "Delivery CTT Express",
+    "summary": "Delivery Carrier implementation for CTT Express API",
+    "version": "16.0.1.0.0",
+    "category": "Delivery",
+    "website": "https://github.com/OCA/delivery-carrier",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": ["delivery_package_number", "delivery_state"],
+    "external_dependencies": {"python": ["zeep"]},
```
Comment (pedrobaeza) [inline delivery_cttexpress/__manifest__.py]: Don't include it, as it's an standard requirement:

https://github.com/odoo/odoo/blob/16.0/requirements.txt#L50

**Review by pedrobaeza — APPROVED** [review]

File: delivery_cttexpress/__manifest__.py (hunk 1 above)
Comment (chienandalu) [reply delivery_cttexpress/__manifest__.py]: Done!

**Review by victoralmau — APPROVED** [review]
Code review OK

**Pull request merged**